apps/test/unit/p5lab/spritelab/SpritelabTest.js

```diff
@@ -145,6 +146,32 @@ describe('SpriteLab', () => {
         expect(resultingAnimations.orderedKeys.length).to.be.equal(1);
       });
 
+      describe('react to executionError', () => {
+        let alertSpy;
+        beforeEach(() => {
+          alertSpy = sinon.stub(studioApp(), 'displayWorkspaceAlert');
+          sinon.stub(instance, 'getMsg').returns({
+            workspaceAlertError: () => {
+              return 'translated string';
+            }
+          });
+        });
+        afterEach(() => {
+          alertSpy.restore();
+          instance.getMsg.restore();
+        });
+        it('displays a workspace alert if there is an executionError message', () => {
+          const msg = 'test string';
+          instance.reactToExecutionError(msg);
```

Comment:
super nit: i think it would actually be more readable to remove `msg` here and pass our test value directly to the function (`instance.reactToExecutionError('test string')`)